rtcdatachannel.go

```diff
@@ -128,10 +128,13 @@ func (api *API) newRTCDataChannel(params *RTCDataChannelParameters) (*RTCDataCha
 	}
 
 	d := &RTCDataChannel{
-		Label:      params.Label,
-		ID:         &params.ID,
-		ReadyState: RTCDataChannelStateConnecting,
-		api:        api,
+		Label:             params.Label,
```

Comment:
Maybe it's a good moment to make these attributes private? We can't control routine safety if they are public. Thoughts @Sean-Der ?

Reply:
That could be perceived as a breaking change tho...

Reply:
Yea, I know. Most of these didn't do anything in the past thought :/

Reply:
I say break them! If people need values we can add thread safe getters for them later, this will all be part of `v2.0.0`

SCTP is getting such a huge upgrade people will totally understand

Reply:
I can do this right now... (I would hide DataChannelParameters also)

Reply:
Done!